paletteData.js

```diff
@@ -0,0 +1,65 @@
+module.exports = [
```

Comment:
The palette and project data looks good. I like the past mod project references too 🙂